### PR TITLE
📦 NEW: VSCode.pro

### PR DESCRIPTION
**Pull request by ahmadawais**
Adding a new site to the list. My new [VSCode Power User](https://VSCode.pro) course site. Thanks a lot, Gatsby for enabling me to do make such an awesome PWA with offline support that didn't cost me months.

Looking forward, peace! ✌️

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -3656,3 +3656,19 @@
...
   built_by_url: https://kylemcd.com
   featured: false
+- title: VSCode Power User Course
+  main_url: https://VSCode.pro
+  url: https://VSCode.pro
+  description: >
+    After 10 years with Sublime, I switched to VSCode. Love it. Spent 1000+ hours building a premium video course to help you switch today. 200+ power user tips & tricks turn you into a VSCode.pro
+  categories:
+    - Education
+    - Course
```
Comment (LekoArts) [inline docs/sites.yml]: IMO better fitting and the tag already exists 😊 

```suggestion
    - Learning
```

Comment (ahmadawais) [reply docs/sites.yml]: Makes a lot more sense. Just committed.

**Comment by ahmadawais** [comment]
@LekoArts thanks for merging it. 

🤔 I read the docs couldn't find how I can submit https://VSCode.pro for review to be added in the featured site. I've built it from scratch, think it's pretty good (some excellent reviews on Twitter).

🌟 As of right now, VSCode.pro is also featured on the homepage of _the_ [OnePageLove.com](https://onepagelove.com/) and here's the [direct link](https://onepagelove.com/vscode-power-user) just in case you read this later.

✅ I'd love to request my site to be featured on the showcase. Slowly and steadily I am going to open source parts of it. Here's the first release of [styled-rmq](https://twitter.com/MrAhmadAwais/status/1070318224426893312).

Looking forward, peace! ✌️

**Comment by KyleAMathews** [comment]
We haven't decided on a process for promoting new sites to "featured". One plan is that we'll have a quarterly community voting on sites to promote the best sites to the showcase but that's as far as we've gotten.

**Comment by ahmadawais** [comment]
@KyleAMathews that makes sense. In my experience community voting contests lead to weird motivations from those who don't play well. Instead, time could be well spent on a #GatsbyWeekly hashtag (Twitter/Insta/LI) where you folks can talk about one thing you like about a featured website. At least, that'd be my suggestion if I were running DevRels sort of stuff at GatsbyJS. 

🌟 As a core WordPress developer, I think Gatsby is the best modern stack replacement — I am thinking about making my next course on something like that. 

Anywho, let me know if there's anything I can do to help to get https://VSCode.pro featured. I'll soon be sharing stuff like — I built it with Gatsby and I'm super happy with how it turned out.

Peace! ✌️

**Comment by KyleAMathews** [comment]
@ahmadawais great feedback — I haven't run one of these before so good to know.

Glad it's been a great experience! Would love to see the course!!

**Comment by ahmadawais** [comment]
@KyleAMathews glad you liked it. Let me know how I can help. I'll be doing a video on Gatsby very soon. 👍

**Comment by ahmadawais** [comment]
Folks is there a way to refresh the image being rendered here? https://www.gatsbyjs.org/showcase/vscode.pro

I think there was a buggy deploy which got cached. Or service workers acting out again.

Looking forward, peace! ✌️

**Pull request merged**